ClientFactory/PluginClientFactory.php

```diff
@@ -2,8 +2,8 @@
 
 namespace Http\HttplugBundle\ClientFactory;
 
-use Http\Client\Plugin\Plugin;
-use Http\Client\Plugin\PluginClient;
+use Http\Client\Common\Plugin;
+use Http\Client\Common\PluginClient;
```

Comment:
Hm, you are right. However I think it is a design mistake to rely on the plugin client: one's code should work with any kind of http client.